apps/main/src/dao/hooks/useProposalsList.ts

```diff
@@ -24,16 +24,20 @@ const sortProposals = (
 ) => {
   if (activeSortBy === 'endingSoon') {
     const currentTimestamp = Math.floor(Date.now() / 1000)
-    const activeProposals = proposals.filter((proposal) => proposal.timestamp + WEEK > currentTimestamp)
-    const passedProposals = orderBy(
-      proposals.filter((proposal) => proposal.timestamp + WEEK < currentTimestamp),
+    const activeProposals = proposals.filter((proposal) => proposal.status === 'Active')
```

Comment:
We could use [lodash `partition`](https://lodash.com/docs/4.17.15#partition) here
```suggestion
    const [activeProposals, inactiveProposals] = partition(proposals, (proposal) => proposal.status === 'Active')
```

Reply:
FYI @OnlyJousting